Sources/RequestBuilder.swift

```diff
@@ -75,6 +75,8 @@ extension ResourceRequest {
         let totalCount: Int?
         if let totalCountString = response.allHeaderFields["X-List-Totalcount"] as? String {
             totalCount = Int(totalCountString)
+        } else if let totalCountString = response.allHeaderFields["x-list-totalcount"] as? String {
+            totalCount = Int(totalCountString)
```

Comment:
I think it is not need to this change because `HTTPURLResponse#allHeaderFields` keys are *case-insensitive*.

https://developer.apple.com/documentation/foundation/httpurlresponse/1417930-allheaderfields

Reply:
I thought that allHeaderFields was case insensitive, but it seems that Swift 3 and later are case sensitive. Perhaps it is necessary to distinguish case for all key values.

https://stackoverflow.com/questions/40152483/httpurlresponse-allheaderfields-swift-3-capitalisation

Reply:
hmm...
OK, Please add comment for this issue in source code. thanks.

Reply:
Sorry for being late.

The Swift bug is troubling us.
I tried convert the all header keys to lower case.
How about this implementation?

https://github.com/slightair/GarageClientSwift/pull/7/commits/7529629685e916f1467fe7b3f910a47b6cb70541#diff-1d696d720be823d13aae2986bcc884f5R76